.env

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
Does docker-compose automatically look for a file named `.env` or did you have to specify that filename somewhere?

I don't see where `index.js` loads `.env`. Where is that?